### DISCUSSION TITLE
copilot.vim error: Copilot: Node.js exited with status 134

### DISCUSSION THREAD

**Discussion by ChrisPenner** [community/community#116678] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hey folks, after a recent neovim upgrade Copilot has stopped working for me in neovim, it used to work great (using https://github.com/github/copilot.vim). Every command, e.g. `:Copilot setup` or `:Copilot status` just errors with:

```
Copilot: Node.js exited with status 134
```

Things I've tried:
* I've tried pinning node to each of `v16.19.0`, `v17.4.0` and `v21.7.1`, and verified the version seen by neovim using `:r! node --version`
* Setting `export NODE_OPTIONS="--max-old-space-size=10096"` in my shell rc; since apparently code 134 is often associated with not having enough heap memory available
* I've tried re-installing Node and copilot.vim

No luck so far, any tips on how to sort this out, or even how to get more information out of copilot to debug things?

Thanks!

**Comment by ChrisPenner** [comment, marked as answer]
Figured it out, but I had to dig through a bunch of source code to figure it out. Would have been nicer to see the actual error output from `node` rather than just the exit code, and also maybe the node executable that Copilot had decided to use.

In my case, I had forgotten I'd hard-coded the version of node copilot used  because it used to only support node 16; this got baked into my Packer config, so even after I removed the config it didn't change because I had to recompile with `PackerCompile`.

Switching to a modern node and removing all hardcoded node versions from my plugin config sorted it out.

**Comment by yokurang** [comment]
Hi, I also faced this issue using the Lazy package manager. For me, I only had to update my Nodejs version to fix this issue.
